pkgs/tools/misc/dylibbundler/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "dylibbundler";
+
+  src  = fetchFromGitHub {
+    owner  = "auriamg";
+    repo   = "/macdylibbundler";
+    rev    = "a59987097d970d45473ae3383cec9c8f78cf556c";
+    sha256 = "13pqzlm6lhbndw1ij3bpkhcl5va519552hfavff7pyifgk6liz6z";
+  };
+
+  installPhase = ''
```

Comment:
We should use the provided install phase:

https://github.com/auriamg/macdylibbundler/blob/master/Makefile#L19